tests/e2e/support/objects/app-admin-settings/users/index.ts

```diff
@@ -9,17 +11,22 @@ export class Users {
     this.#usersEnvironment = new UsersEnvironment()
     this.#page = page
   }
-  getUUID({ key }: { key: string }): string {
-    return this.#usersEnvironment.getCreatedUser({ key }).uuid
+  async getUUID({ key }: { key: string }): Promise<string> {
+    if (config.keycloak) {
+      const user = this.#usersEnvironment.getUser({ key })
+      const admin = this.#usersEnvironment.getUser({ key: 'admin' })
+      return await getUserId({ user, admin })
```

Comment:
will the id be same for the user throughout the single scenario? if yes then maybe a separate use store for keycloak and store respective ids when the user is created?

Reply:
lets do that in separate PR